src/pages/menu/basket/index.js

```diff
@@ -32,6 +36,23 @@ const Basket = () => {
     strings: { basket: translations },
   } = useTranslation();
 
+  const { renderNotification, showNotification } = useNotifications(
+    <Flex direction="column">
```

Comment:
the same as in prev PR, try to create a common component for notification with icon and text. Also, mb it's a good idea to union these bugs and this PR's